docs/user-guide/deploy-on-bare-metal.md

```diff
@@ -38,91 +38,21 @@ _*Note*: This method could be used also in Virtual environment_
-
-# This command will host the iPXE files on an podman container
-podman run  -v ${IPXE_DIR}:/app:ro -p 8080:8080 -d --rm docker.io/bitnami/nginx:latest
-```
-
-To ensure if your container is working fine, check the url with a `curl` command
-
-```shell
-curl http://$(hostname):8080/ipxe
-```
-
-### Manual
-
-The manual way is explained here. You need at least to have the Discovery ISO already generated
-
-Now let's download that ISO in the provisioning machine, where the iPXE files will be hosted (use the _Command to download the ISO_ button from the Assisted Service website
-
-```shell
-IPXE_DIR=/tmp/ipxe/ai
-IMAGE_PATH=/tmp/discovery_image_ocp.iso
-ISO_URL=$(curl http://console.redhat.com/api/assisted-install/v2/infra-envs/<infra_env_id>/downloads/image-url | jq ".url")
-
-wget -O ${IMAGE_PATH} ${ISO_URL}
 ```
-
-- Now we need to create the folder and the _ipxe_ file definition
-
-```shell
-mkdir -p ${IPXE_DIR}
-
-cat > $IPXE_DIR/ipxe << EOF
+$ curl "http://assisted.example.com:8090/api/assisted-install/v2/infra-envs/a7acfb01-d89f-40c8-82d7-02b20cf00173/downloads/files?file_name=ipxe-script"
 #!ipxe
-set live_url $(hostname):8080
-initrd --name initrd \${live_url}/initrd.img
-kernel \${live_url}/vmlinuz initrd=initrd ignition.config.url=\${live_url}/config.ign coreos.live.rootfs_url=\${live_url}/rootfs.img ${KERNEL_OPTS}
+initrd --name initrd http://assisted.example.com:8888/images/a7acfb01-d89f-40c8-82d7-02b20cf00173/pxe-initrd?arch=x86_64&version=4.9
+kernel http://assisted.example.com:8888/boot-artifacts/kernel?arch=x86_64&version=4.9 initrd=initrd coreos.live.rootfs_url=http://assisted.example.com:8888/boot-artifacts/rootfs?arch=x86_64&version=4.9 random.trust_cpu=on rd.luks.options=discard ignition.firstboot ignition.platform.id=metal console=tty1 console=ttyS1,115200n8 coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8"
 boot
-EOF
 ```
 
-- We also need to extract the images from the ISO
-
-```shell
-PXE_IMAGES=$(isoinfo -R -i $IMAGE_PATH -f | grep -i images/pxeboot)
-
-for img in $PXE_IMAGES; do
-  name=$(basename ${img})
-  echo extracting $name
-  isoinfo -R -i $IMAGE_PATH -x $img > $IPXE_DIR/$name
-done
-```
-
-- And as a last step, write the Ignition files for the deployment
-
-```shell
-echo writing custom user ignition
-ziptool=$(isoinfo -R -i $IMAGE_PATH -x /images/ignition.img | file -b - | cut -d ' ' -f 1)
-isoinfo -R -i $IMAGE_PATH -x /images/ignition.img | ${ziptool,,} -dc | cpio -iD $IPXE_DIR
-```
-
-- After the Ignition files creation we need to host the files, for that we will use a podman contianer based on nginx
-
-```shell
-podman run  -v ${IPXE_DIR}:/app:ro -p 8080:8080 -d --rm docker.io/bitnami/nginx:latest
-```
-
-- To ensure if your container is working fine, check the url with a `curl` command
-
-```shell
-curl http://$(hostname):8080/ipxe
-```
+This URL can either be used directly or the script can be downloaded and hosted separately
 
 ### Booting the nodes from iPXE
```

Comment:
I'm not sure if it's worth changing this here.
It's good information about how to set the script url, but the actual url doesn't match with the example I added.

@javipolo @CrystalChun do you think it's worth re-doing the demo/recording for a few screenshots?

Reply:
If we were to redo it then would it be the same flow displayed in the screenshots or would we show a different flow? If it's the same flow, then maybe a note at the top about using correct urls would be sufficient?

Reply:
:man_shrugging: there's a lot of things we could document.
This shows adding the url to the console directly rather than configuring DHCP, but I thought people would generally be using DHCP.

The steps after entering the script address don't seem specific to ipxe though

Reply:
I think that just adding a note should be enough. Something saying that now it's not needed to download the ISO and extract the artifacts, since you can just copy-paste the ipxe script we provide and that's all

Reply:
I think I handled that.
Ideally a user wouldn't even need to take the script, just point their environment to the script endpoint we expose, right?

Reply:
That's correct :)